### DISCUSSION TITLE
UnicodeDecodeError While Running Paraview Example

### DISCUSSION THREAD

**Discussion by ghost** [Kitware/trame#557] [General] [answered]
Hello, I'm trying to run a Paraview simple cone example (https://github.com/Kitware/trame/blob/master/examples/07_paraview/SimpleCone/LocalRendering.py), but it shows UnicodeDecodeError: 'utf-8' codec can't decode byte 0xac in position 1: invalid start byte.

![image](https://github.com/Kitware/trame/assets/93155074/8531c6e0-b811-428d-b27e-3bb59654b2dc)

**Comment by jourdain** [comment]
can you list your package versions by running `pip list`?

**Reply by ghost** [reply]
![image](https://github.com/Kitware/trame/assets/93155074/225fa603-7f07-4b4b-ac7e-c18588d71075)

**Reply by jourdain** [reply, marked as answer]
try running `pip install trame trame-vtk -U`

**Reply by jourdain** [reply]
And then make sure you clear your browser cache or use a different port.

**Reply by ghost** [reply]
It works now. Thank you :)
